tests/ctst/features/replicatateExistingObjects.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Replicate existing objects created before enabling replication
```

Comment:
maybe the "Feature" should be `replication`; and we will later expend it to test live replication as well.